storage/innobase/xtrabackup/test/t/undo_tablespaces.sh

```diff
@@ -79,9 +79,9 @@ xtrabackup --prepare --target-dir=$topdir/backup --incremental-dir=$topdir/inc
 xtrabackup --copy-back --target-dir=$topdir/backup
 
 
-for file in $undo_directory_ext/undo1.ibu $undo_directory_ext/undo2.ibu \
+for file in $undo_directory/undo1.ibu $undo_directory/undo2.ibu \
```

Comment:
difference between undo_directory and undo_directory_ext?

Also this doesn't seem to a testcase. Can you please add one.

Reply:
updated the testcase.